.go-version

```diff
@@ -0,0 +1 @@
+1.17.9
```

Comment:
This is just prep for switching the release process, but it was a small change and generically related to versioning, so I included it.

Reply:
```suggestion
1.17.9

```
Add newline to get rid of the little stopsign 😄